gulp-config.js

```diff
@@ -1,12 +1,12 @@
 module.exports = function () {
-  var instanceRoot = "C:\\websites\\Habitat.dev.local";
+  var instanceRoot = "C:\\projects\\GIT\\SitecoreHabitat\\Websites";
```

Comment:
Please ensure you leave the default values in the files

Reply:
Sure. Changing it back to what it was